src/test/java/persistence/action/ActionQueueTest.java

```diff
@@ -0,0 +1,103 @@
+package persistence.action;
+
+import domain.Person;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.metadata.MetaModel;
+import persistence.sql.db.H2Database;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ActionQueueTest extends H2Database {
+
+    private ActionQueue actionQueue;
+    private MetaModel metaModel;
+
+    private Person kakaoPerson;
+    private Person naverPerson;
+
+    @BeforeEach
+    void setUp() {
+        this.kakaoPerson = new Person(1L, "kakao", 10, "@kakao.com");
+        this.naverPerson = new Person(null, "naver", 11, "@naver.com");
+
+        metaModel = inFlightMetadataCollector.getMetaModel();
+        actionQueue = entityManagerFactory.getActionQueue();
+
+        actionQueue.allExecute();
+        entityManager.removeAll(Person.class);
+    }
+
+    @DisplayName("insertQueue 아이디값이 있을경우에는 Queue에 추가한다.")
+    @Test
+    void addInsertAction() {
+        InsertEntityAction<Person> personInsertEntityAction = new InsertEntityAction<>(metaModel.getEntityPersister(Person.class),
+                kakaoPerson,
+                false);
+
+        actionQueue.addInsertAction(personInsertEntityAction);
+
+        Person person = entityManager.find(Person.class, kakaoPerson.getId());
+
+        assertThat(person).isNull();
```

Comment:
```suggestion
        entityManager.find(Person.class, ID_EXSIST_PERSON.getId());

        InsertEntityAction<?> actual = entityManagerFactory.getActionQueue().insertQueuePoll();
        InsertEntityAction<Person> expected = new InsertEntityAction<>(metaModel.getEntityPersister(Person.class),
                ID_EXSIST_PERSON,
                false);

        assertThat(actual).isEqualTo(expected);
```

작성하신 내부 구현을 제가 100% 이해한것은 아니기때문에 조심스럽긴한데,
혹시 이런 테스트를 생각하신걸까요?
`insertQueue 아이디값이 있을경우에는 Queue에 추가한다.` 의 테스트라면 우선 제가 떠오르는건 "Queue의 상태검증" 인데 그러한 부분이 아닌것 같아서요 🤔 

테스트 코드안에서의 `entityManager.actionQueue`는 테스트코드에서 사용하는 `actionQueue`와 다른객체 일텐데? 라는 의문이 들어요 🤔 
```
InsertEntityAction<Person> personInsertEntityAction = new InsertEntityAction<>(metaModel.getEntityPersister(Person.class),
        kakaoPerson,
        false);
actionQueue.addInsertAction(personInsertEntityAction);
```
위 준비과정이 빠져도 정상 성공을 해요 🤔

Reply:
앗 actionQueue도 확인하는 로직이 빠졌네요
Entitymangerfactory 쪽에서 반환하는거 만들었는데 놓쳤네요